notebooks/245-oneformer-segmentation/245-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        "pixel_values": torch.randn(1, 3, *shape),  # TODO: make shapes dynamic
</pre><p>TODOs are left in code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>